MobileCenterCrashes/MobileCenterCrashes.xcodeproj/project.pbxproj

```diff
@@ -839,17 +839,24 @@
 			name = iOS;
 			sourceTree = "<group>";
 		};
-		350B29E91F192A5C009B91CF /* WrapperSdkUtilities */ = {
+		350B29E91F192A5C009B91CF /* WrapperSDKUtilities */ = {
```

Comment:
Why do we have two groups for WrapperSDKUtilities?

Reply:
One is internal, this other one private ;)